syllabiShare/views.py

```diff
@@ -176,18 +186,33 @@ def schooladmin(request,domain=None):
     return redirect('/')
 
 
+def saved(request):
+    (template, context) = authenticate(request.user)
+    if template:
+        return render(request, template, context)
+
+    if 'unsave' in request.POST:
```

Comment:
Not too comfortable how this logic is duplicated above in display. Might be better if we sent the POSTs to a specific URL?

Reply:
Yeah we can do that I figured that would be a good idea

Reply:
is that really worth it though? with this logic its easy to see what views can remove a saved syllabus. If we did a standalone url then we would need to do a lot of error checking to ensure they arent removing a syllabus that doesnt exist by accidentally refreshing / navigating to the removal url